src/components/Atoms/Input/Input.js

```diff
@@ -92,6 +95,7 @@ const Input = React.forwardRef(
           aria-describedby={hasAria ? id : undefined}
           ref={ref}
           prefixLength={prefix.length}
+          required={optional === false}
```

Comment:
Could move this line above `{...rest}` so that `<Input required>` still works?

Reply:
Ooh yeah, very good call

Reply:
Yeah makes sense